modules/cluster-logging-cpu-memory.adoc

```diff
@@ -61,15 +61,13 @@ spec:
             memory: 100Mi
       replicas: 2
   collection:
-    logs:
-      type: "fluentd"
-      fluentd:
-        resources: <3>
-          limits:
-            memory: 736Mi
-          requests:
-            cpu: 200m
-            memory: 736Mi
+    resources: <3>
+      limits:
+        memory: 736Mi
+      requests:
```

Comment:
Slight indent on the requests section is intended or should it align with `resources`?

![Screenshot from 2024-08-29 10-21-11](https://github.com/user-attachments/assets/f618d69b-f710-4563-b12b-ee6ef102b6f5)

Reply:
Actually, I should align with `limits` which it is as per my observation.

Reply:
Thanks!